### PR TITLE
Added metadata to tile_source.py

### PR DESCRIPTION
**Pull request by AmyyyCao**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [ ] issues: fixes #xxxx
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

**Comment by AmyyyCao** [comment]
Fixing #11765 

Added Metadata to:
```examples/plotting/file/tile_source.py```

File: examples/plotting/file/tile_source.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
+''' This example displays a CartoDB Positron base world map
```
Comment (g-parki) [inline examples/plotting/file/tile_source.py:1]:
```suggestion
''' This example displays a CartoDB Positron base world map.
```

**Comment by g-parki** [comment]
@AmyyyCao I left a couple initial comments on the metadata itself. There are also a couple issues you'll need to work through:

1. There are unrelated `.DS_STORE` files committed to this branch. These will need to be removed in order for the PR to be approved. Just FYI this PR will automatically update itself if you force push changes to your branch - which is what we prefer instead of starting a new PR.
2. There is a lint issue caught by our automatic codebase tests (see output below). I'll paste in a recommendation I made to another contributor:
- Run codebase tests manually: `pytest tests/codebase`
- [Set up pre-commit](https://docs.bokeh.org/en/latest/docs/dev_guide/setup.html#set-up-pre-commit) to run tests automatically before pushing
- See full details for Bokeh tests in the [dev guide](https://docs.bokeh.org/en/latest/docs/dev_guide/testing.html)

Codebase test output:
```
E       AssertionError: Flake8 issues:
82
E         examples/plotting/file/tile_source.py:16:1: E101 indentation contains mixed spaces and tabs
```

**Comment by g-parki** [comment]
Hey @AmyyyCao, checking in. Do you intend to proceed with this PR?

**Comment by bryevdv** [comment]
@AmyyyCao will you be able to continue with this PR? If not we can close it.

**Comment by AmyyyCao** [comment]
@bryevdv Hi, sorry I will not be able to continue with this PR

**Pull request closed without merging**